pkg/reconciler/v1alpha1/revision/resources/deploy.go

```diff
@@ -149,6 +149,8 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, observab
 		Volumes:                       append([]corev1.Volume{varLogVolume}, rev.Spec.Volumes...),
 		ServiceAccountName:            rev.Spec.ServiceAccountName,
 		TerminationGracePeriodSeconds: &revisionTimeout,
+		NodeSelector:                  rev.Spec.NodeSelector,
+		Tolerations:                   rev.Spec.Tolerations,
```

Comment:
Can we confirm these are mutable.

If not we will need changes to the `checkAndUpdateDeployment` logic here:
https://github.com/knative/serving/blob/bbe4c2f6e2d383130a6fb9a802c456a04ec27fc0/pkg/reconciler/v1alpha1/revision/cruds.go#L66-L68